aws-predictions/src/main/java/com/amplifyframework/predictions/aws/http/LivenessWebSocket.kt

```diff
@@ -96,7 +97,7 @@ internal class LivenessWebSocket(
     @VisibleForTesting
     internal var webSocket: WebSocket? = null
     internal val challengeId = UUID.randomUUID().toString()
-    var challengeType: ChallengeType? = null
+    lateinit var challengeType: FaceLivenessChallengeType
```

Comment:
I'd rather leave this nullable rather than relying on a server response to guarantee lateinit behavior.